packages/neutron-sdk/src/query/total_burned_neutrons.rs

```diff
@@ -0,0 +1,20 @@
+use crate::{bindings::query::InterchainQueries, NeutronResult};
+use cosmwasm_std::{Coin, Deps};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct TotalBurnedNeutronsAmountResponse {
+    pub coins: Vec<Coin>,
```

Comment:
Why is it a `Vec<>`?

Reply:
I was not sure how to handle the potential occassion of neutron denom changing, hence I decided to use a vec here.
If neutron denom shall never change, then I can remove this parameter from governance, and store here either sdk.Coin or maybe some kind of integer (which one though? there are uint64, big,Int, sdk.Int).

Reply:
Neutron denom shouldn't be changed, so just a `Coin` is ok